README.md

```diff
@@ -1,169 +1,97 @@
 # kube-spawn
 
-## Introduction
-
-__kube-spawn__ is a tool for creating a multi-node Kubernetes cluster
+`kube-spawn` is a tool for creating a multi-node Kubernetes cluster
 on a single machine, created mostly for developers __of__ Kubernetes.
 
 It aims to be as similar to the solutions recommended for production
 clusters as possible.
 
-## Getting started
+## Requirements
+
+* **Host:**
+  - `systemd-nspawn` at least version 233
+  - tested: Fedora 26, ...
 
-### Build proper systemd-nspawn version
+* **Kubernetes** at least version 1.7.0
 
-Unfortunately, there is [one pending feature](https://github.com/systemd/systemd/pull/4395)
-of systemd-nspawn which is merged in master, but not released yet.
-You will need to build your own systemd-nspawn binary.
+## Quickstart
 
-We highly recommend using CoreOS' fork which backported that feature
-to the 231 version of systemd (which is the one that Fedora and
-the other popular distributions are using in its stable releases).
+To quickly test `kube-spawn`, the easiest (and tested) way is to use the prepared [Vagrant setup](doc/vagrant.md).
 
-In order to do that, please use the following commands:
+To setup `kube-spawn` on your host, make sure you have `glide` available in you PATH.
+In the directory where you cloned this repository, please do:
```

Comment:
I guess this is important for the instructions after `go get ...`, `make vendor all` etc. The directory where I cloned kube-spawn is `cd $GOPATH/src/github.com/kinvolk ; git clone ...` - maybe just say "Switch into the kube-spawn repsitory ..."